### ISSUE TITLE
PrimeNG calendar accessibility not working

### ISSUE THREAD

**Issue by aakashnebhani** [primefaces/primeng#10770]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Stackblitz Case (Bug Reports)**
https://stackblitz.com/edit/primeng-calendar-demo

**Current behavior**
When we use [monthNavigator]="true" for navigation calendar then after changing the month from dropdown the focus keeps moving in the top bar and user is not able to select the date in the new month from the keyboard. This is happening when monthNavigator is kept true. In case monthNavigator is false then it works fine

**Expected behavior**
User should be able to select date by keyboard when month is changed and monthNavigator is true

**Minimal reproduction of the problem with instructions**
1. Open https://stackblitz.com/edit/primeng-calendar-demo
2. Click on Navigators example in date field
3. Change the month to anything other than current month
4. Use keyboard to select the date of new month selected in previous step

**What is the motivation / use case for changing the behavior?**
Users who are dependent on accessibility should be able to work with calendar

**Please tell us about your environment:**
Windows and Mac

* **Angular version:** 5.X
Angular 11.x

* **PrimeNG version:** 5.X
PrimeNG 11.x but the issue is reproducible in PrimeNG 12.x as well

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

* **Node (for AoT issues):** `node --version` =   14.17.1
